server/liveblog/blogs/embeds.py

```diff
@@ -85,13 +85,21 @@ def render_bloglist_embed(api_host=None, assets_root=None):
     return render_template('blog-list-embed.html', **scope)
 
 
-@embed_blueprint.route('/embed/<blog_id>')
-def embed(blog_id, api_host=None, theme=None):
+@embed_blueprint.route('/embed/<blog_id>', defaults={'theme': None, 'output': None})
+@embed_blueprint.route('/embed/<blog_id>/<theme>', defaults={'output': None})
```

Comment:
this is development use only, but the s3 url link is constructed the same.
Theme is added due to multiple theme support and also forward planing for theme support on output.